specs-jetpack-calypso/wp-jetpack-connect-spec.js

```diff
@@ -274,61 +275,52 @@ test.describe( `Jetpack Connect: (${ screenSize })`, function() {
 			return driverManager.ensureNotLoggedIn( driver );
 		} );
 
-		test.it( 'Can register new Subscriber user', () => {
-			this.emailAddress = dataHelper.getEmailAddress( dataHelper.getNewBlogName(), signupInboxId );
+		test.it( 'Can register new Subscriber user', function() {
+			this.blogName = dataHelper.getNewBlogName();
+			this.emailAddress = dataHelper.getEmailAddress( this.blogName, signupInboxId );
 			this.password = config.get( 'passwordForNewTestSignUps' );
-			this.signupFlow = new SignUpFlow( driver, {
+
+			const signupFlow = new SignUpFlow( driver, {
 				emailAddress: this.emailAddress,
+				blogName: this.blogName,
 				password: this.password,
 			} );
-			return this.signupFlow
+			return signupFlow
 				.signupFreePlan()
```

Comment:
Not really relevant to this PR but we don't need to sign up for a free site, we could just create a user account using https://wordpress.com/start/account